### ISSUE TITLE
Format of groundtruth bounding box format

### ISSUE THREAD

**Issue by zhi-dian** [fundamentalvision/Deformable-DETR#230]
Thank you for your awesome work! I'm currently using def detr for customized dataset and get a bit confused on the format of target bbox format. According to loss function, the format should be cxcyhw.
![image](https://github.com/fundamentalvision/Deformable-DETR/assets/104620617/b7e24e3d-aa42-462f-919d-c6e0928f0d03)
But the getitem method of coco dataset is xyxy. 
![image](https://github.com/fundamentalvision/Deformable-DETR/assets/104620617/61b1a63f-f58a-4c39-ab75-e766bd3a1ac4)
Does the format matter given it is an end-to-end set prediction? And if I want to use pretrained weight on coco dataset, which format should I use?
